docs/Makefile

```diff
@@ -8,6 +8,8 @@ SPHINXPROJ    = Optuna
 SOURCEDIR     = source
 BUILDDIR      = build
 
+export PYTHONPATH := $(realpath ..):$(PYTHONPATH)  
+
```

Comment:
Should this change also be separated?

Reply:
I'm wondering if you cannot append `'plotly_figures/plot_contour.html'` to `html_static_path ` https://github.com/optuna/optuna/blob/master/docs/source/conf.py#L101 instead of modifying the `PYTHONPATH`.

Reply:
We can use store the HTML files in the `_static` folder and embed them in `iframe` as follows. In this way, we don't need to modify `PYTHONPATH`.

```diff
        .. raw:: html
-            :file: ../../docs/source/plotly_figures/plot_contour.html
+            <iframe src="../_static/plot_contour.html" width="100%" height="500px" frameborder="0">
+            </iframe>
```

Reply:
Great. I am just wondering what the benefits of using an iframe are. Is it cleaner in some sense (mostly thinking about style properties that you included, since they're not required with `:file:`) or can the file not be found otherwise?

Reply:
> or can the file not be found otherwise?

Correct.

Reply:
As you pointed out, we need to set some attributes of an iframe to render the plot properly.

Reply:
@hvy Which way would you prefer (modifying `PYTHONPATH` or using an iframe)?

Reply:
Sorry for the delayed response. I think it's better not to modify `PYTHONPATH`. 

Just curious what error you get when trying to read the raw HTML file without loading it in an iframe. It seems to work locally at least.

Reply:
```
.. raw:: html
    :file: _static/plot_contour.html
```

gives:

```
InputError: [Errno 2] No such file or directory: '../optuna/visualization/_static/plot_contour.html'.
```

Reply:
I see. As I wrote above, it worked for me locally. But let's proceed with an iframe if that works as well. Could you go ahead and remove the `PYTHONPATH` and introduce the changes discussed in this thread?

Reply:
> it worked for me locally.

Was the plot rendered properly?

> Could you go ahead and remove the PYTHONPATH and introduce the changes discussed in this thread?

Sure.